### PR TITLE
fix: use new temp-privatek8s public ipv4

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

**Comment by lemeurherve** [comment]
Paired with @smerle33

**Review by dduportal — CHANGES_REQUESTED** [review]
At first sight, you don't have the correct IP.

The terraform diff is a good hint:

```
# aws_security_group.ec2_agents_infraci will be updated in-place
  ~ resource "aws_security_group" "ec2_agents_infraci" {
        id                     = "sg-0371c18c6435745f0"
      ~ ingress                = [
          - {
              - cidr_blocks      = [
                  - "20.72.105.159/32",
                ]
              - description      = "Allow SSH from infra.ci"
              - from_port        = 22
              - ipv6_cidr_blocks = []
              - prefix_list_ids  = []
              - protocol         = "tcp"
              - security_groups  = []
              - self             = false
              - to_port          = 22
            },
          + {
              + cidr_blocks      = [
                  + "20.97.148.67/32",
                ]
              + description      = "Allow SSH from infra.ci"
              + from_port        = 22
              + ipv6_cidr_blocks = []
              + prefix_list_ids  = []
              + protocol         = "tcp"
              + security_groups  = []
              + self             = false
              + to_port          = 22
            },
        ]
        name                   = "ec2_agents_infraci"
        tags                   = {
            "jenkins" = "infra.ci.jenkins.io"
        }
        # (7 unchanged attributes hidden)
    }
    ```

**Comment by lemeurherve** [comment]
Took the wrong one of these:
image

File: ec2-agents.tf [hunk 1]
```diff
@@ -83,7 +83,7 @@ resource "aws_security_group" "ec2_agents_infraci" {
     to_port     = 22
     protocol    = "tcp"
     cidr_blocks = [
-      "52.177.88.13/32" # AKS cluster outside IP
+      "20.97.148.67/32" # temp-privatek8s AKS cluster outside IP
```
Comment (lemeurherve) [inline ec2-agents.tf:86]:
```suggestion
      "20.72.105.159/32" # temp-privatek8s AKS cluster outside IP
```

**Review by dduportal — APPROVED** [review]

**Pull request merged**